pep-9999.rst

```diff
@@ -47,13 +47,57 @@ required to interpret all standard constructs, and additionally a type checker
 can support extra constructs not described in this PEP. All deviations from
 this PEP should be documented.
 
+Syntax and Directory Structure
+==============================
+
+Type stubs are syntactically valid Python files with a ``.pyi`` suffix.
+Type checkers are expected to support the syntax of the latest Python
+version. Type stubs should be parseable with the latest released Python
+version, but also support the previous version up until six months after
+the release of the current version. Older Python versions do not need to be
+supported, even if the implementation supports older versions or the
+type checker is run with an older version.
+
+For example, it is recommended to use PEP 526 [#pep526]_ variable
+annotations, introduced in Python 3.6, in all stubs, even for Python 2-only
+modules.
+
+Type stub files can either live next to their respective implementation
+files or in a separate directory::
```

Comment:
nit: extra colon

Reply:
The extra colon is ReStructuredText syntax to introduce a preformatted block.